network-api/networkapi/wagtailpages/utils.py

```diff
@@ -495,6 +495,11 @@ def get_research_authors(profiles: "QuerySet[Profile]") -> "QuerySet[Profile]":
     return profiles.filter(authored_research__isnull=False).distinct()
 
 
+def get_rcc_authors(profiles: "QuerySet[Profile]") -> "QuerySet[Profile]":
```

Comment:
**suggested/thought**

Looks like this was flagged already that `filter_` probably makes more sense for this method name,
but it might be worth considering the context of the method a bit. I know opinions are split about that
but considering the wide use of it, it might be more helpful to read this method as:

```
filter_rcc_authors_by_articles_authored()
```

Reply:
Yep, this one needs some more thought, but that makes sense, thanks!